packages/client/src/workflow-client.ts

```diff
@@ -747,12 +781,10 @@ export class WorkflowClient {
         });
       },
       async describe() {
-        return this.client.service.describeWorkflowExecution({
-          namespace,
-          execution: {
-            runId,
-            workflowId,
-          },
+        const next = this.client._describeWorkflowHandler.bind(this.client);
+        const fn = interceptors.length ? composeInterceptors(interceptors, 'describe', next) : next;
+        return await fn({
+          workflowExecution: { workflowId, runId },
```

Comment:
I took out `firstExecutionRunId` because it isn't used in the grpc call

Reply:
It will later but we can leave it out for now